js/src/components/contact-information/phone-number-card/index.js

```diff
@@ -1,26 +1,19 @@
 /**
  * External dependencies
  */
-import {
-	getCountryCallingCode,
-	parsePhoneNumberFromString as parsePhoneNumber,
-} from 'libphonenumber-js';
 import { __ } from '@wordpress/i18n';
 import { useState, useEffect } from '@wordpress/element';
-import { Flex, FlexItem, FlexBlock, CardDivider } from '@wordpress/components';
+import { CardDivider } from '@wordpress/components';
 import { Spinner } from '@woocommerce/components';
 
 /**
  * Internal dependencies
  */
-import useCountryCallingCodeOptions from '.~/hooks/useCountryCallingCodeOptions';
-import Section from '.~/wcdl/section';
-import SelectControl from '.~/wcdl/select-control';
-import AppInputControl from '.~/components/app-input-control';
 import AccountCard, { APPEARANCE } from '.~/components/account-card';
 import AppButton from '.~/components/app-button';
 import AppSpinner from '.~/components/app-spinner';
-import './phone-number-card.scss';
+import EditPhoneNumberContent from './edit-phone-number-content.js';
+import './index.scss';
```

Comment:
:nail_care:  If we would like to make some baby steps towards the new naming convention, https://github.com/woocommerce/woocommerce-admin/pull/7505 (p90Yrv-2k8-p2)
WE could keep it as `'./phone-number-card.scss';`

Reply:
Thanks for informing me of that! I renamed **phone-number-card/index.js** to **phone-number-card/phone-number-card.js** and for **phone-number-card/index.scss** in the same way.